gst/nnstreamer/tensor_query/tensor_query_client.c

```diff
@@ -15,6 +15,36 @@
 #endif
 
 #include "tensor_query_client.h"
+#include <gio/gio.h>
+#include <gio/gsocket.h>
+
+/**
+ * @brief Macro for debug mode.
+ */
+#ifndef DBG
+#define DBG (!self->silent)
+#endif
+
+/**
+ * @brief Properties.
+ */
+enum
+{
+  PROP_0,
+  PROP_SINK_HOST,
+  PROP_SINK_PORT,
+  PROP_SRC_HOST,
+  PROP_SRC_PORT,
+  PROP_PROTOCOL,
+  PROP_SILENT,
+};
+
+#define TCP_HIGHEST_PORT        65535
+#define TCP_DEFAULT_HOST        "localhost"
+#define TCP_DEFAULT_SINK_PORT        3000
+#define TCP_DEFAULT_SRC_PORT        3001
```

Comment:
It's good for this version.

In future PR, please make default PORT common for both client and server (in the nnstreamer conf header). And let it be adjusted by .ini and envvar by adding entry in nnstreamer.ini and nnstreamer-conf.

Reply:
Acked.